smelli/test_classes.py

```diff
@@ -55,6 +55,12 @@ def test_incl_excl(self):
         self.assertNotIn('likelihood_lfv.yaml', set(ll.keys()))
         self.assertRaises(ValueError, GlobalLikelihood, include_likelihoods=["nonexistent_likelihood.yaml"])
         self.assertRaises(ValueError, GlobalLikelihood, exclude_likelihoods=["nonexistent_likelihood.yaml"])
+        self.assertRaises(ValueError, GlobalLikelihood, add_measurements={"nonexistent_likelihood.yaml":""})
+        self.assertRaises(ValueError, GlobalLikelihood, remove_measurements={"nonexistent_likelihood.yaml":""})
+        self.assertRaises(ValueError, GlobalLikelihood, add_measurements={"likelihood_ewpt.yaml":"nonexistent_measurement"})
+        self.assertRaises(ValueError, GlobalLikelihood, remove_measurements={"likelihood_ewpt.yaml":"nonexistent_measurement"})
```

Comment:
@Jorge-Alda your test cases use wrong input types. The values of the dictionaries should be lists of strings, not strings. So this results in an error, but not the one tested for.

Reply:
Another case for starting to use type hints :laughing: